### ISSUE TITLE
About optimizer setting in Iterative Alignment

### ISSUE THREAD

**Issue by davidluciolu** [RLHF-V/RLAIF-V#19]
As you mentioned in Implementation Details, you train the model for 4 epoches with learning rate 5e-7.

I want to know that in each round of DPO, the optimizer is newly created or just load from the latest checkpoint?

For example, in the epoch 2, is the learning rate reset to 5e-7? Or it continues from the end of the epoch 1?

**Comment by Haoye17** [comment]
Hi @davidluciolu, thank you for your interest in our work!

We do not reset the learning rate in each training epoch. In a single iteration that contains 4 epochs, we use 5e-7 as the peak learning rate with a 5% warm-up ratio and a cosine scheduler. And for each iteration, the optimizer will be newly created.

If you have any other questions, we are willing to help!

**Comment by davidluciolu** [comment]
Thanks for your reply!
